NoPawsLeftBehind/Controllers/AnimalController.cs

```diff
@@ -53,6 +53,12 @@ public async Task<IActionResult> CreateAnimal([FromBody]CreateAnimal animal)
             await Db.Connection.OpenAsync();
             Console.WriteLine(animal);
 
+            AnimalQuery addQuery = new AnimalQuery(Db);
+
+            await addQuery.addAnimal(animal.Type, animal.Breed, animal.Gender, animal.Name,
```

Comment:
One recommendation I would make here would be to pass the CreateAnimal object directly to the addAnimal query rather than adding each property as a parameter. It's generally best practice to keep the number of function parameters to 3~4 max if possible.

So the addAnimal function would look like:

`public async Task AddAnimal (CreateAnimal animal) 
{
    ...
    apiHelper.BindIntParam(cmd, Tuple.Create("@typeID", animal.typeID));
    ...
`

I'll still merge this into main because it's not functionally wrong, just styling preferences. Looks good otherwise